Library/Homebrew/cask/dsl.rb

```diff
@@ -431,14 +432,22 @@ def livecheck(&block)
       @livecheck ||= Livecheck.new(cask)
       return @livecheck unless block
 
-      if !@cask.allow_reassignment && @livecheckable
+      if !@cask.allow_reassignment && @livecheck_block
         raise CaskInvalidError.new(cask, "'livecheck' stanza may only appear once.")
       end
 
-      @livecheckable = true
+      @livecheck_block = true
       @livecheck.instance_eval(&block)
     end
 
+    # Whether the cask contains a `livecheck` block. This is a legacy alias
+    # for `#livecheck_block?`.
+    sig { returns(T::Boolean) }
+    def livecheckable?
+      odeprecated "`livecheckable?`", "`livecheck_block?`"
+      @livecheck_block == true
+    end
```

Comment:
As it stands, I've kept the `#livecheckable?`methods with an `odeprecated` call, to ease this transition. I've replaced all usage in this repository but I'm not sure if maintainers/contributors or third-parties have used `#livecheckable?` outside of this.

As one example, I have some scripts that I run (locally) that uses parts of livecheck and a few of those have `#livecheckable?` calls that I'll need to update. I also vaguely remember some third-party doing something similar years ago (I can't remember the project) and it only came to my attention when they were bitten by a similar change.

I'm not against simply replacing `#livecheckable?` with `#livecheck_block?` but going through a deprecation process may help folks to transition their code.